File: JavaAndroid/app/src/main/java/java/sample/Apt.java
package java.sample;

import android.content.Context;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.TextView;

import java.util.List;

/**
 * Created by SidneyXu on 2016/01/19.
 */
public class Apt extends ArrayAdapter<String> {
    public Apt(final Context context, List<String> data) {
        super(context, android.R.layout.simple_list_item_1, android.R.id.text1, data);
    }

    @Override
    public String getItem(final int position) {
        return super.getItem(position);
    }

    @Override
    public int getCount() {
        return super.getCount();
    }

    @Override
    public View getView(final int position, final View convertView, final ViewGroup parent) {
        View view = super.getView(position, convertView, parent);
        TextView title = (TextView) view.findViewById(android.R.id.text1);
        title.setText(getItem(position));
        return view;
    }
}

File: KotlinAndroid/app/src/main/kotlin/kotlin/sample/Apt.kt
package kotlin.sample

import android.content.Context
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.TextView

/**
 * Created by SidneyXu on 2015/11/30.
 */
public class Apt(val ctx: Context, var data: List<String>) : ArrayAdapter<String>(
        ctx,
        android.R.layout.simple_list_item_1,
        android.R.id.text1,
        data) {

    override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View? {
        val view = super.getView(position, convertView, parent)
        val title = view.findViewById(android.R.id.text1) as TextView
        title.text = getItem(position)
        return view
    }
}